////////////////////////////////////////////////////////////////////////////////////////////////////
///
/// @file      AudioFormat.proto
///
/// @brief     This Protocol Buffer contains information from SHARC DSP about the channel
//             configuration and incoming audio format.
///
/// @attention Copyright (C) 2017 Bose Corporation All Rights Reserved
///
///            Bose Corporation
///            The Mountain Road,
///            Framingham, MA 01701-9168
///            U.S.A.
///
///            This program may not be reproduced, in whole or in part, in any form by any means
///            whatsoever without the written permission of Bose Corporation.
///
////////////////////////////////////////////////////////////////////////////////////////////////////
package ProductPb;

message AudioFormat
{
    optional string channels = 1;
    optional string format   = 2;
}

////////////////////////////////////////////////////////////////////////////////////////////////////
///                                        End of File                                           ///
////////////////////////////////////////////////////////////////////////////////////////////////////
